elliptic-curve/src/arithmetic.rs

```diff
@@ -88,10 +88,10 @@ pub trait PrimeCurveArithmetic:
 /// Normalize point(s) in projective representation by converting them to their affine ones.
 pub trait BatchNormalize<Points>: group::Curve {
     /// The output of the batch normalization; a container of affine points.
-    type Output;
+    type Output: AsRef<Self::AffineRepr>;
```

Comment:
Try:

```rust
type Output: AsRef<[Self::AffineRepr]>;
```

...but it would also have to be the following below:

```rust
fn batch_normalize(points: &Points) -> <Self as BatchNormalize<Points>>::Output;
```

Reply:
Perfect this is working. https://github.com/RustCrypto/elliptic-curves/pull/971 should be good now too